src/plan_ordered_append.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+#ifndef TIMESCALEDB_PLAN_ORDERED_APPEND_H
+#define TIMESCALEDB_PLAN_ORDERED_APPEND_H
+
+#include <postgres.h>
+#include <nodes/relation.h>
+
+#include "hypertable.h"
+#include "hypertable_restrict_info.h"
+
+bool		ts_ordered_append_do_optimization(PlannerInfo *root, RelOptInfo *rel, Hypertable *ht, bool *reverse);
+void		ts_add_ordered_append_path(PlannerInfo *root, RelOptInfo *rel, Hypertable *ht, MergeAppendPath *merge);
+Path	   *ts_ordered_append_path_create(PlannerInfo *root, RelOptInfo *rel, Hypertable *ht, MergeAppendPath *merge);
+List	   *ts_ordered_append_path_find_children(HypertableRestrictInfo *hri, Hypertable *ht, LOCKMODE lockmode, bool reverse);
+
+typedef struct TimescaleDBPrivate
```

Comment:
This should be in the main planner file and not here.

Reply:
moved to planner.h